webhook_to_fedora_messaging/models/util.py

```diff
@@ -0,0 +1,64 @@
+# SPDX-FileCopyrightText: Contributors to the Fedora Project
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+import datetime
+
+from sqlalchemy import Column, UnicodeText
+from sqlalchemy.ext.compiler import compiles
+from sqlalchemy.sql.expression import FunctionElement
+from sqlalchemy.types import DateTime as SQLDateTime
+from sqlalchemy.types import TypeDecorator
+
+from uuid import uuid4
+
+
+class utcnow(FunctionElement):
+    """
+    Current timestamp in UTC for SQL expressions
+    """
+    type = SQLDateTime
+    inherit_cache = True
+
+
+class TZDateTime(TypeDecorator):
```

Comment:
If this class isn't used anymore, let's remove it. As I mentioned before I think we should use plain DateTime types and use timezone-aware datetimes everywhere.